droidlet/perception/semantic_parsing/nsp_transformer_model/train_model_clean.py

```diff
@@ -0,0 +1,737 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
why do we need a new file ? can we name this better to reflect the purpose of the file ?

Reply:
@kavyasrinet 
I was not sure if I should clean up the previous training script without making sure that new retraining pipeline is working. I named train_model_clean just to indicate it this is the cleaned-up version of training script. If it looks fine, I will replace the previous one with this one and remove it.